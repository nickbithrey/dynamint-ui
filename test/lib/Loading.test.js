import React from 'react';
import { shallow } from 'enzyme';

import Loading from '~/lib/Loading';

describe('<Loading /> rendering', () => {

	it('renders a loading text', () => {
		const wrapper = shallow( <Loading /> );
		
		expect(wrapper).toMatchSnapshot();
	});
	
		
});