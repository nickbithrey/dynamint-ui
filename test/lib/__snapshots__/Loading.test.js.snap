// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loading /> rendering renders a loading text 1`] = `
<p>
  Loading
</p>
`;
